config/watcher.ts

```diff
@@ -22,6 +22,9 @@ const config: Config = {
         "core/client/auth/**/*.ts",
         "core/client/auth/**/*.tsx",
         "core/client/auth/**/*.graphql",
+        "core/client/admin/**/*.ts",
+        "core/client/admin/**/*.tsx",
+        "core/client/admin/**/*.graphql",
```

Comment:
Not sure how the `auth` lines got here, but `npm run compile:relay-stream` does not touch these files. We can delete lines 22-27.